broadcast/001_DeployPluginManager.s.sol/1/run-17364640803.md

```diff
@@ -0,0 +1,3 @@
+- This chain used cast send to deploy so there's no corresponding run json
```

Comment:
For future refactoring, I’m considering consolidating this file with the CMD file to streamline the structure. What are your thoughts?

Reply:
Yes open to consolidation. Could you explain more the new strucutre you propose for the future change?

Reply:
Rather than splitting it into two files, we could include the cast command directly in this file. For example:

```
command: cast send xxx
txHash: 0xabc
```